scripts/ci/docker-compose.yml

```diff
@@ -85,17 +88,14 @@ services:
       - TRAVIS_REPO_SLUG
       - TRAVIS_OS_NAME
       - TRAVIS_TAG
+      - RUN_TESTS
+      - AIRFLOW_CI_VERBOSE
     depends_on:
-      - postgres
-      - mysql
       - mongo
       - cassandra
       - rabbitmq
       - redis
       - openldap
       - krb5-kdc-server
-    volumes:
-      - ../../:/app
-      - ~/.cache/pip:/home/airflow/.cache/pip
-      - ~/.wheelhouse/:/home/airflow/.wheelhouse/
-    working_dir: /app
+      - mysql
+      - postgres
```

Comment:
Minor point: could we put these two lines back to their original order+position?

Reply:
Sure! will do.

Reply:
Reverted in latest commit.